src/OptiHPLCHandler/empower_api_core.py

```diff
@@ -115,47 +133,85 @@ def logout(self) -> None:
         self.session_id = None
         logger.debug("Logout successful")
 
-    def get(self, endpoint: str) -> requests.Response:
+    def _requests_wrapper(
+        self, method: str, endpoint: str, body: Optional[dict], timeout
+    ) -> requests.Response:
         """
-        Get data from Empower.
+        Wrapper for requests.
 
-        :param endpoint: The endpoint to get data from.
+        :param method: The method to use.
+        :param endpoint: The endpoint to use.
+        :param body: The body to use.
         """
```

Comment:
MIssing parameter in doc-string.

Reply:
Another question is whether to even have a docstring, it is an internal function. But now it is there, so I might as well keep it.